code/src/components/subintro/Subintro.tsx

```diff
@@ -0,0 +1,23 @@
+import "@/components/subintro/subintro.scss";
+import subintroData from "@/components/subintro/Subintro.constants";
+import SubintroElement from "@/components/subintroElement/SubintroElement";
+import PageWrapper from "@/layouts/app-wrapper/PageWrapper";
+import Box from "@mui/material/Box";
```

Comment:
move this import to the top of file, because it's import from library

Reply:
Done